src/routes/game/lobby/validation.ts

```diff
@@ -22,7 +22,6 @@ export const deckValidation = (
         }, 0);
         return numHands <= heroNumHands;
       })
-      .min(1, 'Pick at least one weapon/off-hand.')
```

Comment:
I think in this case it would be more prudent to have an explicit “no weapon” that can be selected, much like for armour. I can see people deselecting their weapon, or not having it automatically selected, by accident and then complaining about it.